lib/API.jsx

```diff
@@ -3,12 +3,12 @@
  * WIP, This is in the process of migration from web-e. Please add methods to this as is needed.|
  * ----------------------------------------------------------------------------------------------
  */
-import _ from 'underscore';
 
 // Use this deferred lib so we don't have a dependency on jQuery (so we can use this module in mobile)
 import {Deferred} from 'simply-deferred';
+import {has} from 'lodash';
```

Comment:
maybe I'm misinterpreting the issue goal but shouldn't we be removing anything from lodash?

Reply:
Oh sorry, I forgot to update the title of the PR and its description. We decided to remove all these libraries only inside files that we will be using in `react-native-live-markdown` and execute on the UI thread (`ExpensiMark`, `Src` etc). Also, we decided to move more complex `underscore` functions to `lodash`, like during migration to TS in E/App, because it doesn't make sense to rewrite them on our own in the utils file

Reply:
I updated the PR description